<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlVariablesDialog</class>
 <widget class="QWidget" name="ControlVariablesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Variables</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Operand</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QSpinBox" name="spin_constant">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radio_random">
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radio_item">
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="radio_enemy">
        <property name="text">
         <string>Enemy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_constant">
        <property name="text">
         <string>Constant</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="radio_character">
        <property name="text">
         <string>Character</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_variable">
        <property name="text">
         <string>Variable</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radio_actor">
        <property name="text">
         <string>Actor</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="radio_other">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spin_random_from">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spin_random_to">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="SwitchVariableWidget" name="widget_var" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="combo_item">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>In Inventory</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="combo_actor">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="combo_actor_param">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Level</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EXP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MaxHP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MaxSP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AGI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PDEF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDEF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVA</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="combo_enemy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>1.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8.</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="combo_enemy_param">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>HP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MaxHP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MaxSP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AGI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PDEF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDEF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVA</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="combo_character">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="combo_character_param">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Map X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Map Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Direction</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Screen X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Screen Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Terrain Tag</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="combo_other">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Map ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Party Members</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Steps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Timer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Save Count</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Variable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_single">
        <property name="text">
         <string>Single</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_batch">
        <property name="text">
         <string>Batch</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spin_from">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spin_to">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="SwitchVariableWidget" name="widget_variable" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_set">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radio_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radio_sub">
        <property name="text">
         <string>Sub</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_mul">
        <property name="text">
         <string>Mul</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radio_div">
        <property name="text">
         <string>Div</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radio_mod">
        <property name="text">
         <string>Mod</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchVariableWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/switchvariablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radio_single</tabstop>
  <tabstop>radio_batch</tabstop>
  <tabstop>spin_from</tabstop>
  <tabstop>spin_to</tabstop>
  <tabstop>radio_set</tabstop>
  <tabstop>radio_add</tabstop>
  <tabstop>radio_sub</tabstop>
  <tabstop>radio_mul</tabstop>
  <tabstop>radio_div</tabstop>
  <tabstop>radio_mod</tabstop>
  <tabstop>radio_constant</tabstop>
  <tabstop>spin_constant</tabstop>
  <tabstop>radio_variable</tabstop>
  <tabstop>radio_random</tabstop>
  <tabstop>spin_random_from</tabstop>
  <tabstop>spin_random_to</tabstop>
  <tabstop>radio_item</tabstop>
  <tabstop>combo_item</tabstop>
  <tabstop>radio_actor</tabstop>
  <tabstop>combo_actor</tabstop>
  <tabstop>combo_actor_param</tabstop>
  <tabstop>radio_enemy</tabstop>
  <tabstop>combo_enemy</tabstop>
  <tabstop>combo_enemy_param</tabstop>
  <tabstop>radio_character</tabstop>
  <tabstop>combo_character</tabstop>
  <tabstop>combo_character_param</tabstop>
  <tabstop>radio_other</tabstop>
  <tabstop>combo_other</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_constant</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_constant</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_var</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_single</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_variable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_batch</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_from</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_batch</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_to</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_random</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_random_from</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_random</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_random_to</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_item</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_item</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_actor</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_actor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_actor</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_actor_param</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_enemy</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_enemy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_enemy</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_enemy_param</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_character</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_character</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_character</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_character_param</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_other</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_other</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
